fpdf/fpdf.py

```diff
@@ -2029,28 +2069,43 @@ def cell(
                 "ignored"
             )
             border = 1
+        newpos_x = XPos.RIGHT
+        newpos_y = YPos.TOP
+        if ln == 1:
```

Comment:
Would you like to take this opportunity to introduce `new_x` / `new_y` as arguments to `cell()`,
and add a deprecation warning about `ln`?

Reply:
Since this is a very noticeable change to the public API with widespread consequences throughout documentation and tests, I had planned to do that in a seperate PR, together with obsoleting the `center=False` parameter.

Reply:
Good thinking!